docs/write-a-plugin.md

```diff
@@ -108,4 +108,5 @@ Current version: <span id='tip-version'>loading</span>
 
 <script>
 document.getElementById('tip-version').innerText = Docsify.version
+document.getElementsByClassName("lang-js")[2].innerHTML = document.getElementsByClassName("lang-js")[2].innerHTML.replace(/Last modified .*'/,"Last modified {docsify-updated<span>}'</span>")
```

Comment:
This seems to reverse the breaking effect. Cab we instead stop the breaking in the first place? It can be easier to understand that way, instead if adding "Band-Aids" that make it more difficult to understand what is happening.

Where is the break happening?

Reply:
https://docsify.js.org/#/write-a-plugin?id=edit-button

Reply:
My "band aid" expression maybe sounded too harsh. It is a term my co-workers and I have used to describe quick fixes that don't solve the main issue, but add a workaround instead,  making the code less ideal for the short term gain.

Do you know what is causing the issue in the first place?

Reply:
Because it was replaced

https://github.com/docsifyjs/docsify/blob/407e4d4f3de78bebd639a3fdae751f8045728e57/src/core/render/index.js#L34-L43